process.py

```diff
@@ -82,6 +82,39 @@ def process_csv(filename: str) -> list[dict]:
 
     booklet_only_events = orientation_events
 
+    # Check for conflicts with mandatory events and invalid events
+    mandatory_events = list(
+        orientation_event
+        for orientation_event in orientation_events
+        if "mandatory" in orientation_event["tags"]
+    )
+    for event in api_response["events"]:
+        event_start = datetime.datetime.fromisoformat(event["start"])
+        event_end = datetime.datetime.fromisoformat(event["end"])
+
+        if event_end < event_start:
+            event["tags"].append("invalid")
+            print("Invalid event found: " + event["name"])
+            continue
+
+        for mandatory_event in mandatory_events:
```

Comment:
Could we use the `mandatory` tag to check whether an event is worth throwing a conflict over?

Reply:
yaur that's what used i believe

> ```python
> mandatory_events = list(
>     orientation_event
>     for orientation_event in (orientation_events)
>     if "mandatory" in orientation_event["tags"]
> )
> ````

ill make it also use `api_response["events"]` just in case tho